projects/observability/src/shared/dashboard/widgets/topology/topology-widget.model.ts

```diff
@@ -71,6 +73,13 @@ export class TopologyWidgetModel {
   @ModelInject(MODEL_API)
   private readonly api!: ModelApi;
 
+  @ModelInject(FeatureStateResolver)
+  private readonly featureStateResolver!: FeatureStateResolver;
+
+  public isFeatureEnabled(feature: ApplicationFeature): Observable<boolean> {
```

Comment:
This should not be in the model. The model should have a layout property like we discussed. Similar to showBrush

```
  @ModelProperty({
    key: 'showBrush',
    displayName: 'Show Brush',
    type: BOOLEAN_PROPERTY.type,
    required: false
  })
  public showBrush?: boolean = true;
  ```

Reply:
Then the place where we are sending topology model there we need to put the FF logic.

Reply:
I added it as a config in topology component as that's the one exposed to the outer world. And created a property in TopologyConfiguration

Reply:
Updated